### PR TITLE
GREEN-20334 Updated patch offers and assessment API intro

### PR DESCRIPTION
**Pull request by adabrooker**

### CODE REVIEW

File: source/includes/assessment/_introduction.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ To begin the integration process, please send the following information to us at
 3. URL for your `test_status` API call
 4. A sample API key
 
-We'll add your integration to our [Partners page](http://www.greenhouse.io/partners), add your FAQ article to our help section, and provide you with a Sandbox account to test the integration.
+The Partnerships team will configure the assessment and provide next steps on receiving access to a Sandbox account to test the integration. For any third party assessment partner building an integration to use with mutual customers, it is required to provide documentation on the integration for the Greenhouse Help Center.
```
Comment (robnormal) [inline source/includes/assessment/_introduction.md]: should that be "configure the integration"? "Configure the assessment" doesn't make sense

Comment (robnormal) [reply source/includes/assessment/_introduction.md]: Also, what does that second sentence mean? This was written by someone on the Partnerships team, right?

Comment (adabrooker) [reply source/includes/assessment/_introduction.md]: I updated the wording here.

File: source/includes/assessment/_introduction.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ To begin the integration process, please send the following information to us at
 3. URL for your `test_status` API call
 4. A sample API key
 
-We'll add your integration to our [Partners page](http://www.greenhouse.io/partners), add your FAQ article to our help section, and provide you with a Sandbox account to test the integration.
+The Partnerships team will configure endpoint URLs for creating assessment stages and provide next steps on receiving access to a Sandbox account to test the integration. For any third party assessment partner building an integration to use with mutual customers, it is required to provide documentation on the integration for the Greenhouse Help Center.
```
Comment (robnormal) [inline source/includes/assessment/_introduction.md]: Could that second sentence be, "If you are building an integration for use by mutual customers, we will need documentation on the integration for the Greenhouse Help Center."

**Pull request merged**